snforge_std/src/events.cairo

```diff
@@ -102,21 +103,10 @@ fn assert_if_emitted(ref self: EventSpy, event: Event) -> bool {
         }
 
         if event_name_hash(event.name) == *self.events.at(j).name
-            && event.from == *emitted_events.at(j).from {
-            if @event.keys != emitted_events.at(j).keys
-                || @event.data != emitted_events.at(j).data {
-                panic(
-                    array![
-                        event.name,
-                        'event was emitted from',
-                        event.from.into(),
-                        'but keys or data are different'
-                    ]
-                );
-            }
-
+          && event.from == *emitted_events.at(j).from
+          && @event.keys == emitted_events.at(j).keys
+          && @event.data == emitted_events.at(j).data {
```

Comment:
I liked the old one because it was more descriptive but this way is good too.
We can consider adding a new method to the `EventAssertions` trait: `assert_names_emitted` which would check if there are events with specific names